.github/workflows/README.md

```diff
@@ -0,0 +1,49 @@
+# GitHub Actions CI
+
+## Actions
+ - `custom-actions/build-packager`:
+   Builds Shaka Packager.  Leaves build artifacts in the "artifacts" folder.
+   Requires OS-dependent and build-dependent inputs.
+ - `custom-actions/test-packager`:
+   Tests Shaka Packager.  Requires OS-dependent and build-dependent inputs.
+ - `custom-actions/build-docs`:
+   Builds Shaka Packager docs.
+
+## Workflows
+ - On PR:
```

Comment:
Does it apply to non-release commits too?

It seems to be redundant for PRs, but it will be useful for commits submitted by the internal Gerrit (the test configs for the Gerrit is not exactly the same as the test configs here).

Reply:
We can run workflows on push of new commits, too, if we wish.  It can be the same workflow we use for PRs, or it can be an independent workflow with many of the same steps.  Let's discuss and decide if we want to do this or not.

Reply:
And related to this: the build status needs to be updated: https://github.com/google/shaka-packager/blame/master/README.md#L11. I don't know if GitHub Actions provide a similar feature.

Reply:
GitHub Actions does offer badges.  Having build and test status badges for ToT would suggest that we should have a build-and-test workflow run on pushes, too.  I'll take care of that.